pkg/nfd-worker/nfd-worker.go

```diff
@@ -762,10 +762,14 @@ func (m *nfdWorker) updateNodeFeatureObject(labels Labels) error {
 		nfrUpdated.Labels = map[string]string{nfdv1alpha1.NodeFeatureObjNodeNameLabel: nodename}
 		nfrUpdated.OwnerReferences = ownerRefs
 		nfrUpdated.Spec = nfdv1alpha1.NodeFeatureSpec{
-			Features: *features,
+ 			Features: *features,
 			Labels:   labels,
 		}
-
+		currentTime := time.Now()
+		msg := fmt.Sprintf("Updated: %q", currentTime)
+		nfrUpdated.Status = nfdv1alpha1.NodeFeatureStatus{
+			Status: []string{ msg },
```

Comment:
If we're going to store only one string, why are we storing it in an array here?

Reply:
@TessaIO

Thanks  for the input. Added:

- number of features
- number of labels discovered
- number of sources that had failures

Please take a look.